src/storage/txn/store.rs

```diff
@@ -90,6 +92,7 @@ impl SnapshotStore {
             lower_bound,
             upper_bound,
             self.isolation_level,
+            false,
```

Comment:
Could we use `use_prefix_seek` in `scanner`?

Reply:
We can't.